s3-inf-ppo/laboratoria/lab02/lab02a.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+public class Lab02a
+{
+    static Random rand = new Random();
+    const int students_count = 10;
+
+    public static void Main(string[] args)
+    {
+        List<string> students = new List<string>();
+
+        for (int i = 0; i < students_count; i++)
+        {
+            students.Add(GetRandomStudentNumber());
+        }
+
+        Console.WriteLine("Students group have been filled.");
+
+        for (int i = 0; i < students.Count; i++)
+        {
+            Console.WriteLine(students[i]);
+        }
+        Console.ReadKey();
+    }
+
+    protected static string GetRandomStudentNumber() => (rand.Next(2000) + 38000).ToString();
```

Comment:
Oh, c'mon, these are simple programs for students that just started learning what is an object. Please keep it simple and make it a normal function.